.gitignore

```diff
@@ -1,8 +1,7 @@
 *.iml
 .gradle
-/local.properties
-/.idea/workspace.xml
```

Comment:
Did you remove those intentionally?
According to [JetBrains](https://intellij-support.jetbrains.com/hc/en-us/articles/206827587-How-to-manage-projects-under-Version-Control-Systems) this line should be kept.